components/arm/universalrobots/ur.go

```diff
@@ -427,7 +428,8 @@ func (ua *URArm) MoveToJointPositionRadians(ctx context.Context, radians []float
 			return err
 		}
 		for idx, r := range radians {
-			if math.Round(r*100) != math.Round(state.Joints[idx].Qactual*100) {
+			// blend * 1000 = 1
+			if math.Abs(r-state.Joints[idx].Qactual)*100 > 1 {
```

Comment:
Can we add a comment specifying what we're actually doing here? We are looking to see if we are near the end of the movement to send the next one, right?

Reply:
Could you make the comment more specific? Something like "keep checking here until we are within X distance to the goal", clarifying radians vs mm vs whatever, and describe what happens next and why. e.g. "don't send all positions all at once because the ur5 will discard them" or w/e. The comments here should make it so that no one in the future has to re-do your experimentation with how blend radii work :)

Reply:
todo

Reply:
as written this will prevent us from changing theta supposing all other joints stay in place. TODO: fix